assets/pages/digital-subscription-landing/components/independentJournalismSection.scss

```diff
@@ -0,0 +1,98 @@
+.component-independent-journalism {
+  margin: -1px;
```

Comment:
This seems to be causing a bit of overlap with the footer, and it's popping in horizontal scrollbars in a couple of places 🤔

![margin](https://user-images.githubusercontent.com/5131341/41287951-6bc00b98-6e3c-11e8-9b7f-ef52f5460f79.png)

Reply:
Let me take a look I think that can probably go.